sqlite/src/func.c

```diff
@@ -24,6 +24,8 @@
 #include "comdb2.h"
 #include "sql.h"
 #include "bdb_int.h"
+#include "md5.h"
```

Comment:
Enclose the changes in WITH_SSL ?

Reply:
This is SQLite's MD5 implementation, not openssl's.  We use it in other places (eg: fingerprint code), even without WITH_SSL.

Reply:
oh ok. I looked at the signature thought they were openssl's.